modules/search/instant-search/components/search-box.jsx

```diff
@@ -3,28 +3,34 @@
 /**
  * External dependencies
  */
-import { h, Component } from 'preact';
+import { h } from 'preact';
+import { useState } from 'preact/hooks';
 import { __ } from '@wordpress/i18n';
+// eslint-disable-next-line lodash/import-scope
+import uniqueId from 'lodash/uniqueid';
```

Comment:
This seems to be breaking the build:

```
[04:08:12] 'react:master' errored after 17 s
[04:08:12] ModuleNotFoundError in plugin "webpack"
Message:
    Module not found: Error: Can't resolve 'lodash/uniqueid' in '/home/travis/build/Automattic/jetpack/modules/search/instant-search/components'
```

Reply:
Strange, I'm not able to repro this failure. 

I tried deleting `node_modules`, running `yarn`, and finally running `yarn build && yarn watch`.

Reply:
Ah - I'm guessing you're on a case-sensitive file system @jeherve and it should be `uniqueId` not `uniqueid`. I'll update it now 👍